hack/test-templates.sh

```diff
@@ -93,25 +109,29 @@ esac
 limactl rm -f "${NAME}-tmp"
 
 if [[ -n ${CHECKS["port-forwards"]} ]]; then
-	tmpconfig="$HOME/lima-config-tmp"
+	tmpconfig="$HOME_HOST/lima-config-tmp"
 	mkdir -p "${tmpconfig}"
 	defer "rm -rf \"$tmpconfig\""
 	tmpfile="${tmpconfig}/${NAME}.yaml"
 	cp "$FILE" "${tmpfile}"
 	FILE="${tmpfile}"
 	INFO "Setup port forwarding rules for testing in \"${FILE}\""
 	"${scriptdir}/test-port-forwarding.pl" "${FILE}"
-	limactl validate "$FILE"
+	if [ "$(uname -o)" = "Msys" ]; then
+		limactl validate "$(cygpath -w "$FILE")"
```

Comment:
Maybe you should set `$FILE_HOST` as in `HOME_HOST`

Reply:
Your comment lead to another discovery. I didn't think of it before. It would be more correct to always pass OS dependent path format into limactl and use unix like for tooling. So, I added FILE_HOST, where the platform specific representation of FILE resides and used it through the whole shell file, where appropriate.